_health-care/_js/_components/_form-elements/ErrorableInput.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A form input with a label that can display error messages.
+ *
+ * Validation has the following props.
+ * `errorMessage` - Error string to display in the component.
+ *                  When defined, indicates input has a validation error.
+ * `label` - String for the input field label.
+ * `placeholder` - placeholder string for input field.
+ * `required` - boolean. Render marker indicating field is required.
+ * `value` - string. Value of the input field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableInput extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('validated-input-');
+  }
+
+  handleChange(domEvent) {
+    this.props.onValueChange(domEvent.target.value);
+  }
+
+  render() {
+    // Calculate error state.
+    let errorSpan = '';
+    let errorSpanId = undefined;
+    if (this.props.errorMessage) {
+      errorSpanId = `${this.inputId}-error-message`;
+      errorSpan = <span className="usa-input-error-message" id={`${this.inputId}-error-message`}>{this.props.errorMessage}</span>;
```

Comment:
I think you can reference `errorSpanId` here for the `id` instead of using `${this.inputId}-error-message`